recipes/gdal/pre_3.5.0/conanfile.py

```diff
@@ -434,8 +386,7 @@ def _patch_sources(self):
             # os.path.join("ogr", "ogrsf_frmts", "cad", "libopencad"), # TODO: uncomment when libopencad available
             os.path.join("ogr", "ogrsf_frmts", "geojson", "libjson"),
         ]
-        if Version(self.version) >= "3.1.0":
-            embedded_libs.append(os.path.join("ogr", "ogrsf_frmts", "flatgeobuf", "flatbuffers"))
+        embedded_libs.append(os.path.join("ogr", "ogrsf_frmts", "flatgeobuf", "flatbuffers"))
```

Comment:
embedded_libs is declared as a list a few lines above, can probably move this new item directly into the declaration.

Reply:
thanks !